### PR TITLE
Updating ci-kubernetes-e2e-capz-master-windows-alpha to set more feature gates and run VPA tests

### PR DESCRIPTION
**Pull request by marosset**
/sig windows
/assign @aravindhp @jsturtevant

### CODE REVIEW

File: config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml [hunk 1]
```diff
@@ -445,7 +445,7 @@ periodics:
     preset-azure-cred-only: "true"
     preset-azure-anonymous-pull: "true" # Sets REGISTRY which is needed when building CCM/CNM images
     preset-capz-windows-common: "true"
-    preset-capz-containerd-1-7-latest: "true"
+    preset-capz-windows-2022: "true"
```
Comment (aravindhp) [inline config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml:448]: Was this change intended?

File: config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml [hunk 2]
```diff
@@ -477,12 +477,16 @@ periodics:
...
+          - name: API_SERVER_FEATURE_GATES
+            value: "InPlacePodVerticalScaling=true"
           - name: GINKGO_FOCUS
-            value: (\[Feature:NodeLogQuery\]|\[sig-windows\]) # run the features and a few windows related tests
+            value: (\[Feature:InPlacePodVerticalScaling\]|\[Feature:NodeLogQuery\]|\[sig-windows\]) # run the features and a few windows related tests
           - name: NODE_FEATURE_GATES
-            value: "NodeLogQuery=true"
-          - name: WINDOWS_SERVER_VERSION
-            value: "windows-2022"
+            value: "InPlacePodVerticalScaling=true,NodeLogQuery=true,RuntimeClassInImageCriApi"
```
Comment (aravindhp) [inline config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml:485]: Shouldn't it be `RuntimeClassInImageCriApi=true`?

Comment (jsturtevant) [reply config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml:485]: can we add PodAndContainerStatsFromCRI=true

Comment (jsturtevant) [reply config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml:485]: https://github.com/kubernetes/test-infra/blob/650a3943d3e33a28eda27d0d686b1e3dda2437b9/config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml#L166

Comment (marosset) [reply config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml:485]: whoops, yes

File: config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml (hunk 1 above)
Comment (marosset) [reply config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml:448]: yea, i updated the env vars so we'd use the **nightly** containerd package instead of the 1.7.x ones

File: config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml (hunk 2 above)
Comment (marosset) [reply config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml]: updated (added the =true fo the RUntimeClassINIMageCriApi and added the PodAndContainerStatsFromCRI=true feature gates)

**Review by aravindhp — COMMENTED** [review]
LGTM

File: config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml (hunk 1 above)
Comment (jsturtevant) [reply config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml:448]: see it now, thanks!
